### ISSUE TITLE
There is no activation after the 2nd Conv in each decoder block

### ISSUE THREAD

**Issue by JiamingJiao** [keyu-tian/SparK#63]
https://github.com/keyu-tian/SparK/blob/00883b885dce68413ea048c6703a82d8a67a83b8/pretrain/decoder.py#L26C10-L26C10
I would like to know if I missed something. Is it a typo or designed for some reasons?
Thanks :D

**Comment by keyu-tian** [comment]
@JiamingJiao yes it is by design. I refer to the MobileNet v2 architecture since they recommend for less activations. But i just found [some unet implementation](https://github.com/usuyama/pytorch-unet/blob/master/pytorch_unet.py) uses the second act. I didn't try this but maybe it can be better than the current decoder.
